mobile/android/app/src/main/res/layout/air_qo_circular_widget.xml

```diff
@@ -0,0 +1,100 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/circular_widget_bg"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <RelativeLayout
+        android:id="@+id/circular_index_color"
+        android:layout_width="180dp"
+        android:layout_height="180dp"
+        android:layout_marginStart="30dp"
+        android:layout_marginEnd="30dp"
+        android:background="@drawable/green_circle">
+
+
+        <TextView
+            android:id="@+id/circular_pm_scale"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentTop="true"
+            android:layout_centerInParent="true"
+            android:fontFamily="@font/roboto_mono_medium"
+            android:paddingTop="5dp"
+            android:paddingBottom="10dp"
+            android:text="@string/pmScale"
+            android:textAppearance="@style/pmScaleCircle" />
+
+        <TextView
+            android:id="@+id/circular_pm_value"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:fontFamily="@font/roboto_mono_medium"
```

Comment:
To reduce the app download size, Could we try exploring usage of downloadable fonts. 

https://github.com/android/user-interface-samples/tree/main/DownloadableFonts